edx-platform/pearson-vue-theme/lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -61,20 +64,22 @@ <h3 class="hd hd-3">${_("Reports")}</h3>
       </p>
     %endif
 
-    <div class="mb-15 problems">
+    <p>
+      <div class="problems">
         ${static.renderReact(
           component="ProblemBrowser",
-          id="react-problem-report",
+          id="react-block-listing",
           props={
             "courseId": course.id,
-            "excludeBlockTypes": ['html', 'video', 'discussion'],
```

Comment:
Are these changes part of the Certprep changes? @sergivalero20

Reply:
@Squirrel18 It was present in Juniper Version.